Stabilize randomly failing TestPersistentSession unit test

```
 === RUN   TestPersistentSession
    session_test.go:39: 
        	Error Trace:	session_test.go:39
        	Error:      	Received unexpected error:
        	            	rpc error: code = Unauthenticated desc = error in extracting the client PK: rpc error: code = Unauthenticated desc = invalid access token: token is invalid
        	Test:       	TestPersistentSession
--- FAIL: TestPersistentSession (0.03s)
FAIL
```

--- a/pkg/rpc/client/client_test.go
+++ b/pkg/rpc/client/client_test.go
@@ -93,7 +93,7 @@ var (
 )
 
 func TestMain(m *testing.M) {
-	conf := server.Setup{Network: "unix", Address: address}
+	conf := server.Setup{Network: "unix", Address: address, SessionDurationMins: 1}
 	// forcing session on the configuration
 	conf.RequireSession = true
 	// create the GRPC server here

